blur/rand.py

```diff
@@ -384,78 +384,35 @@ def weighted_choice(weights):
         return random.choice([opt[0] for opt in options])
 
 
-def weighted_shuffle(weights):
+def weighted_order(weights):

```

Comment:
"sort" makes me think that we are sorting relative to the values which appear in the final list. We are sorting, but not exactly. Maybe...